packages/docs/src/components/DocSiteNavigation.tsx

```diff
@@ -88,8 +96,8 @@ const DocSiteNavigation = ({ location }: DocSiteNavProps) => {
   const data = useStaticQuery(graphql`
     query SiteNavQuery {
       allFile(
-        sort: { fields: [relativeDirectory, name] }
         filter: { ext: { eq: ".mdx" }, relativeDirectory: { ne: "not-in-sidebar" } }
```

Comment:
I think it would be valuable, but I'm not sure it is possible. Because we are using a component query and the `useStaticQuery` hook, no variables are able to be used inside the static query